emcc.py

```diff
@@ -1634,6 +1632,42 @@ def get_final():
         options.pre_js = src = options.post_js = None
         if DEBUG: save_intermediate('pre-post')
 
+      if not shared.Settings.SIDE_MODULE:
```

Comment:
I think doing this here is a good change, but I also worry it might break things. We should be careful to run all tests before landing.